100A #1 Introduction to Jupyter Notebooks (3).ipynb

```diff
@@ -0,0 +1,132 @@
+{
```

Comment:
<p>This seems good.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jimstigler/jupyter/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>